### PR TITLE
Implement device reset command

### PR DESCRIPTION
**Pull request by Zifzaf**
Check the following:

 - [ ] Breaking changes marked in commit message
 - [x] Changelog updated
 - [x] Code style cleaned (ran `make style-fix`)
 - [x] Tested on actual hardware

### CODE REVIEW

File: svm40/svm40.c [hunk 1]
```diff
@@ -102,3 +102,7 @@ svm40_get_version(struct svm40_version_information* version_information) {
 
     return NO_ERROR;
 }
+
+int16_t svm40_reset(void) {
+    return sensirion_i2c_write_cmd(SVM40_I2C_ADDRESS, SVM40_CMD_DEVICE_RESET);
```
Comment (rnestler) [inline svm40/svm40.c:107]: Should we wait the 100ms after the reset? I'd at least add a `#define` with the 100ms constant to the header file.

Comment (abrauchli) [reply svm40/svm40.c:107]: For the SPS30 we don't wait either on reset.. just making the constant public is fine IMO

**Review by abrauchli — CHANGES_REQUESTED** [review]
..we're getting closer to the perfect commit though 👍

File: svm40/svm40.c (hunk 1 above)
Comment (Zifzaf) [reply svm40/svm40.c:107]: Added constant in the header file. Is this okay? Anything else?

File: svm40/svm40.c [hunk 2]
```diff
@@ -102,3 +102,7 @@ svm40_get_version(struct svm40_version_information* version_information) {
 
     return NO_ERROR;
 }
+
+int16_t svm40_reset(void) {
```
Comment (rnestler) [inline svm40/svm40.c:106]: I'd keep the function name as close to the datasheet as possible. So I'd call it `device_reset` instead of just `reset`.

**Review by rnestler — COMMENTED** [review]
Except for the naming it LGTM :rainbow:

File: svm40/svm40.c (hunk 1 above)
Comment (abrauchli) [reply svm40/svm40.c:156]: no, this is resolved.

**Review by abrauchli — APPROVED** [review]

**Pull request merged**